internal/dao/stubs/pubkey_dao.go

```diff
@@ -27,15 +28,42 @@ func getPubkeyDao(ctx context.Context) (dao.PubkeyDao, error) {
 	return getPubkeyDaoStub(ctx)
 }
 
-func (mock *PubkeyDaoStub) Create(ctx context.Context, pk *models.Pubkey) error {
-	mock.store = append(mock.store, pk)
+func (stub *pubkeyDaoStub) Create(ctx context.Context, pk *models.Pubkey) error {
+	pk.ID = stub.lastId + 1
```

Comment:
Why are we changing the pk id and didn't we do that before?
I guess we want the pk ids to match our store order?